Makefile

```diff
@@ -223,13 +223,19 @@ webhook:
 	@echo installing cert-manager
 	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml
 	@echo "wait until pods are up"
-	kubectl wait deployment -n cert-manager cert-manager --for condition=Available=True --timeout=90s
-	kubectl wait --for=condition=Ready pod -l app.kubernetes.io/instance=cert-manager -n cert-manager --timeout=30s 
+	-kubectl wait deployment -n cert-manager cert-manager --for condition=Available=True --timeout=180s
+	-kubectl wait --for=condition=Ready pod -l app.kubernetes.io/instance=cert-manager -n cert-manager --timeout=180s 
```

Comment:
Why ignore the possible errors from these lines?

Reply:
I did because of mininum, false case but I think it is better to remove that Thanks!